packages/angular/docs/storybook-configuration-examples.md

```diff
@@ -26,7 +26,7 @@ There are a number of other options available. Let's take a look at some example
 nx g @nx/angular:storybook-configuration ui --tsConfiguration=true
 ```
 
-This will generate a Storybook configuration for the `ui` project using TypeScript for the Storybook configuration files (the files inside the `.storybook` directory).
+This will generate a Storybook configuration for the `ui` project using TypeScript for the Storybook configuration files (the files inside the `.storybook` directory). By default, starting Nx 16, Storybook v7 is used.
```

Comment:
> This will generate a Storybook configuration for the `ui` project using TypeScript for the Storybook configuration files

I think `Storybook configuration` repeated twice doesn't sound natural. Maybe like:

```This will generate Storybook configuration files for the `ui` project in TypeScript (i.e. `.ts` instead of `.js`).```

> By default, starting Nx 16, Storybook v7 is used

What does this mean? That starting in Nx 16, SB v7 means `tsConfiguration` is always true? Can we reword this?

Reply:
I fixed the description to clarify! Thanks!